### PR TITLE
Skip scheduled actions if token is absent

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

**Comment by ShimShtein** [comment]
@adamruzicka, your opinion would be much appreciated. The request is quite simple: I want to skip planning an action, if the token setting is empty.

File: app/services/foreman_rh_cloud/remediations_retriever.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ def initialize(hit_remediation_pairs, logger: Logger.new(IO::NULL))
     end
 
     def create_playbook
+      unless cloud_auth_available?
+        logger.debug('Cloud authentication is not available, cannot continue')
+        return nil
```
Comment (adamruzicka) [inline app/services/foreman_rh_cloud/remediations_retriever.rb:17]: The `nil` is not really needed

**Review by adamruzicka — APPROVED** [review]
This is the approach I'd take if I were to do this myself, lgtm

**Comment by ShimShtein** [comment]
Removed the `nil`

**Review by laviro — APPROVED** [review]
Thanks @ShimShtein ! LGTM

**Pull request merged**